kythe/proto/BUILD

```diff
@@ -681,6 +681,11 @@ proto_library(
     deps = [":storage_proto"],
 )
 
+go_kythe_proto(
+	proto = ":metadata_proto",
```

Comment:
This will require running `kythe/proto/generate_protobufs.py`

Reply:
I ran it and received this error:

Traceback (most recent call last):
  File "***/kythe/proto/generate_protobufs.py", line 101, in <module>
    do_lang(lang='go', ext='pb.go')
  File "***/kythe/proto/generate_protobufs.py", line 62, in do_lang
    rule_dir, proto = rule.lstrip('/').rsplit(':', 1)
TypeError: a bytes-like object is required, not 'str'

Reply:
Ensure that you are running with Python 2, as you will get that error if you run with Python 3

Reply:
Thanks! It worked with Python2.